### PR TITLE
feat(home): config timezone column (#142)

### LINKED ISSUE #142: Config timezone column
## Describe this problem
The agent will collect the log as "2023-02-20 11:29:21 Hello World" while the log has the formatted timestamp ahead of line, but miss matching log such as "Y|2023-02-20 11:31:21|6" which will be dropped.
And the config for timezone column does not work correctly.
 
## To Reproduce
See above
## Additional Information
None

### PR DESCRIPTION
**Pull request by zzhb101**
# Which issue does this PR close?

Closes #142 

# Rationale for this change
 

# What changes are included in this PR?

# Are there any user-facing changes?

# How does this change test

### CODE REVIEW

File: server/home/home-service/src/main/java/io/holoinsight/server/home/biz/common/GaeaSqlTaskUtil.java [hunk 1]
```diff
@@ -235,13 +182,31 @@ public static From buildFrom(List<LogPath> logPaths, LogParse logParse, List<Fil
...
+          // name: 时间(yyyy-MM-dd HH:mm:ss##默认)
+          if (!StringUtils.isEmpty(splitCol.name)) {
+            String timeAndZone = splitCol.name.substring(3, splitCol.name.length() - 1);
+            String[] cols = timeAndZone.split("##");
+            if (cols.length != 2) {
+              continue;
+            }
+            if (timeZones.contains(cols[1])) {
+              timeParse.setTimezone(cols[1]);
+            } else if (!"默认".equals(cols[1])) {
```
Comment (xzchaoo) [inline server/home/home-service/src/main/java/io/holoinsight/server/home/biz/common/GaeaSqlTaskUtil.java:196]: Use English

**Review by xzchaoo — COMMENTED** [review]
Add some comments for changed/added codes.

File: server/home/home-service/src/main/java/io/holoinsight/server/home/biz/common/GaeaSqlTaskUtil.java [hunk 2]
```diff
@@ -235,13 +182,31 @@ public static From buildFrom(List<LogPath> logPaths, LogParse logParse, List<Fil
...
-      TimeConfig timeConfig = logParse.getTimeConfig();
-      if (timeConfig != null && timeConfig.getElect() != null) {
-        timeParse.setElect(timeConfig.getElect());
-        timeParse.setFormat(timeConfig.getFormat());
-        timeParse.setType(timeConfig.getType());
-        timeParse.setLayout(timeConfig.getLayout());
-        timeParse.setTimezone(timeConfig.getTimezone());
+      for (CustomPluginConf.SplitCol splitCol : splitCols) {
+        if ("TIME".equals(splitCol.colType)) {
+          // name: 时间(yyyy-MM-dd HH:mm:ss##默认)
```
[the remark this answers is not in the document]
Comment (zzhb101) [reply server/home/home-service/src/main/java/io/holoinsight/server/home/biz/common/GaeaSqlTaskUtil.java]: removed

**Review by xzchaoo — APPROVED** [review]
LGTM

**Pull request merged**